vscode/oso/client/src/extension.ts

```diff
@@ -17,12 +17,17 @@ import {
   TransportKind,
 } from 'vscode-languageclient/node';
 
+import {
```

Comment:
I think we need to implement opt out:

From https://code.visualstudio.com/docs/getstarted/telemetry

> If you would prefer to not utilize the npm module, it is still recommended that extension authors respect the user's choice by utilizing the isTelemetryEnabled and onDidChangeTelemetryEnabled API.

Reply:
Ignore this, I See #1410